src/pages/payment/credit-card/mobile/mobile.service.ts

```diff
@@ -1,37 +0,0 @@
-import { FormModel } from 'src/core/form/useForm/useForm.types';
```

Comment:
If this file is shared between mobile and desktop, it should be moved to upper directory.

`credit-card.service.ts` instead of `mobile.service.ts`